xml/net/client/protocol.xml

```diff
@@ -162,7 +162,11 @@
     <packet family="Character" action="Remove">
         <comment>Confirm deleting character from an account</comment>
         <field name="session_id" type="short"/>
-        <field name="character_id" type="int"/>
+        <field name="character_id" type="int">
+            <comment>
+                The official client encodes a variable-sized integer with a minimum size of two bytes.
```

Comment:
We could flesh this out a bit more.
"The official client sends a short, which gets written as a variable-sized integer (2-4 bytes) due to a quirk of the official encoding routine.
However, the official server expects an int."

Reply:
Done. I opted to squash the changes (along with the change in casing for "Official") in with the existing commits to make the history less noisy.